cpp/CMakeLists.txt

```diff
@@ -20,10 +20,17 @@ set(SOURCE_FILES src/ext/dubins.cpp src/ext/dubins.h src/ext/json.hpp src/core/s
         src/vns/plan.h src/vns/fire_data.h src/vns/neighborhoods/moves.h src/vns/neighborhoods/shuffling.h src/vns/neighborhoods/smoothing.h
         src/dubins3d.h  src/core/structures/trajectories.h src/core/updates/updates.cpp src/core/updates/updates.h src/utils.cpp src/vns/factory.cpp src/vns/factory.h)
 
-add_executable(tests ${SOURCE_FILES} src/test/core/test_reversible_updates.h src/test/test_dubins.h
-        src/test/test_position_manipulation.h src/test/main_tests.cpp)
-
 pybind11_add_module(uav_planning ${SOURCE_FILES} src/python_interface.cpp)
 
 set(PLANNING_MODULE_NAME "uav_planning.cpython-35m-x86_64-linux-gnu.so")
 set(PLANNING_MODULE "${CMAKE_CURRENT_BINARY_DIR}/${PLANNING_MODULE_NAME}" PARENT_SCOPE)
+
+
+find_package(Boost COMPONENTS unit_test_framework REQUIRED)
```

Comment:
Is it possible to only require boost for the "tests" target. I know this is usually trivial to do in java related build tools but I don't know about CMake.
If it is not very easy, then just leave it like this.

Reply:
I am a complete noob in CMake so I haven't found yet how to do it. However I will try to find out a solution.